src/lib/autoLauncher.ts

```diff
@@ -63,6 +63,14 @@ Icon=PATH_TO_APP_ICON
     const autostartDirectory = (await path.configDir()) + "/autostart"
     const autostartAppFile = autostartDirectory + "/" + appName + ".desktop"
     return autostartAppFile
+  },
+  async createAutostartDir(appName: string): Promise<string> {
+    const autostartDirectory = (await path.configDir()) + "autostart"
```

Comment:
Above function has `/` on this line and this one doesn't. They shouldn't be both correct at the same time.

Reply:
I am currently working on merge `tauri-upgrade` to this branch. Will be fixed on the next push.  ThanksI